### PR TITLE
Makes admin ui more straightforward to use

### PR DESCRIPTION
**Pull request by hwbllmnn**

### CODE REVIEW

**Comment by weskamm** [comment]
Looking nice, please merge at will

File: app/view/grid/UserListModel.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ Ext.define('MoMo.admin.view.grid.UserListModel', {
     alias: 'viewmodel.momo-userlist',
 
     data: {
-        title: 'All users',
+        title: 'All sers',
```
Comment (marcjansen) [inline app/view/grid/UserListModel.js]: typo?

File: app/view/grid/UserListModel.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ Ext.define('MoMo.admin.view.grid.UserListModel', {
         deleteUserText: 'Do you really want to delete the selected ' +
             'account(s)?',
         deletionSuccessText: 'An account has been successfully deleted',
-        deletionFailureText: 'An account could not be deleted'
+        deletionFailureText: 'An account could not be deleted',
+        userlistFirstNameLabel: ''
```
Comment (marcjansen) [inline app/view/grid/UserListModel.js]: Is this empty for a reason? If this is to hide the label, there is also a config, AFAIR

**Review by marcjansen — APPROVED** [review]
LGTM, only minors

File: app/view/grid/UserListModel.js (hunk 1 above)
Comment (hwbllmnn) [reply app/view/grid/UserListModel.js]: The data will be overridden by the values from the i18n files anyway, this was just a leftover from a test (fixed it anyway).

File: app/view/grid/UserListModel.js (hunk 2 above)
Comment (hwbllmnn) [reply app/view/grid/UserListModel.js]: See the other comment, i18n stuff (added a placeholder anyway).

**Comment by weskamm** [comment]
thanks, merging now

**Pull request merged**